pkg/reconciler/trigger/trigger.go

```diff
@@ -196,14 +198,19 @@ func (r *Reconciler) reconcile(ctx context.Context, t *v1alpha1.Trigger) error {
 		return err
 	}
 
-	trackAddressable := r.addressableTracker.TrackInNamespace(t)
-	if t.Spec.Subscriber != nil && t.Spec.Subscriber.Ref != nil {
-		if err := trackAddressable(*t.Spec.Subscriber.Ref); err != nil {
-			return fmt.Errorf("unable to track changes to Subscriber.Ref: %v", err)
+	if t.Spec.Subscriber == nil {
+		return fmt.Errorf("Subscriber cannot be nil")
+	}
+
+	if t.Spec.Subscriber.Ref != nil {
+		// To call URIFromDestination(dest apisv1alpha1.Destination, parent interface{}), dest.Ref must have a Namespace
+		// If there is no Namespace defined in dest.Ref, we will use the Namespace of Trigger as the Namespace of dest.Ref
+		if t.Spec.Subscriber.Ref.Namespace == "" {
```

Comment:
As far as I can tell, historically we did not allow Namespace to be set here. When would we expect this to not be the empty string?

Reply:
I'm not sure. It looks like the previous code always used the Trigger's namespace, so we should probably do that here.